### ISSUE TITLE
Performance in iOS Safari

### ISSUE THREAD

**Issue by ksmithut** [asvd/intence#1]
Great UX idea! I love it when people challenge the usage of those types of "norms".

Performance on iOS Safari (8.2 on iPhone 6+) is pretty janky. Plus, it's a little jarring to have scrollable elements in the middle of the page take over scrolling, but that happens anyway on normal scrolling elements.

**Comment by asvd** [comment]
Thx for the feedback!

The performace issue on the webpage of intence may also be caused by some additional scripts used there. I will investigate on it.

Not sure what you mean on "scrollable elements in the middle". You mean the menu on the left side coming over the content, or the intence indicator on top and bottom?

**Comment by ksmithut** [comment]
Ah yes, sorry for not clarifying. So there are elements on the page that are scrollable outside of the main window, like the side-scrolling scroll or the map. When scrolling down on the phone, if I start the swipe up by touching down on one of those elements, it stop scrolling the page, and starts scrolling that "sub" element. But, that issue exists with the default scrolling behavior, even without your library. Disregard that issue :)

**Comment by asvd** [comment]
yep, you are right: the library hides the scrollbars, but leaves all the standard features of scrollable areas, along with their issues

**Comment by haraldox** [comment]
try CSS property for iOS safari smooth scrolling:

`.scroll-container {
  -webkit-overflow-scrolling:touch;
}`

**Comment by MarcusJT** [comment]
It's a great effect that I'd love to use it for a scrollable tab bar but alas it's pretty janky on Chrome for Android too, and in Chrome for OSX, which alas rules it out for the time being, which is a shame...

Perhaps check http://jankfree.org/ and see if there's some quick wins?
